### ISSUE TITLE
BUG  This module causes the glitches with the combat tracker

### ISSUE THREAD

**Issue by MotoMoto1234** [YourFavouriteOreo/VTT-Chat-GIFs#13]
This module causes any hidden monsters in the combat tracker in dnd5e to be hidden from the DM also in the list.  

![Chatgif bug](https://user-images.githubusercontent.com/116308065/219684438-fbcc936f-a58a-42e6-a84e-ae2eb04c4092.gif)

Also there is a Custom CSS module override to fix the module covering up chat:

```js
#gifSearch {
    right: 320px;
    bottom: 50px;
    background: #000A !important;
    padding: 5px;
    border-radius: 10px;
}

#gifSearch input {
    background: #60606060 !important;
    border: 1px solid #404040;
    margin: 5px 6px;
    width: 96%;
}

#gifSearch .attribution {
    height: 10px;
    opacity: 50%;
}
```

**Comment by Asinphi** [comment]
Should be fixed with version 1.1.0.

**Comment by MotoMoto1234** [comment]
Yep, updated the module yesterday looks great!
